build.properties

```diff
@@ -2,10 +2,8 @@
 # Don't directly modify this file. Instead, copy it to local.build.properties and
 # edit that.
 #
-# $Id$
 project.name = eXist-db
-project.version = 3.0
```

Comment:
well, as @dizzzz wrote, if we use project.version instead of project.version.numeric, we won't have a way to indicate something like "3.2.0RC" because the semver check does a > or < comparison. So this would mean to either introduce a new property or leave the old two properties, sorry :-(